sdk/servicebus/service-bus/samples-v1/javascript/advanced/sessionRoundRobin.js

```diff
@@ -0,0 +1,170 @@
+/*
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
+
+  This sample demonstrates how you can continually read through all the available
+  sessions in a Service Bus queue or subscription.
+
+  Run the sendMessages sample with different session ids before running this sample.
+*/
+
+const {
+    ServiceBusClient,
+    delay,
+    ReceiveMode,
+  } = require("@azure/service-bus");
+  const dotenv = require("dotenv");
+  const { env } = require("process");
+  const { AbortController } = require("@azure/abort-controller");
+  
+  dotenv.config();
+  
+  const serviceBusConnectionString =
+    env["SERVICEBUS_CONNECTION_STRING"] || "<service bus connection string not in environment>";
```

Comment:
ENV file and the other samples have `SERVICE_BUS_CONNECTION_STRING`.
Let's meet the local consistency at the samples-level before we go for the consistency with the tests ?

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/8833/commits/d58aa56fe56401bf4b246798ea5e78e4891d56d3